src/components/Quote.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './Quote.css';
+
+function Quote() {
+  return (
+    <div>
+      <h1>Math Magician</h1>
+      <p className="quote">
+        Mathmatics is not about numbers ,equations computation, or algorithm
+        its about understanding.
+        <br />
+        <br />
+        {' '}
+        - William paul Thurston
+      </p>
+    </div>
+
+  );
+}
+
```

Comment:
- [ ] The Quote component that should be used ought to be the one you built during the _**Fetch data**_ milestone. Kindly use that component and not this component that renders a static quote. 👍 😎